platform/mellanox/sdk.mk

```diff
@@ -1,4 +1,4 @@
-MLNX_SDK_BASE_URL = https://github.com/Mellanox/SAI-Implementation/raw/310eb922e133ecb3243e10efa4440fabe0ca73c7/sdk
+MLNX_SDK_BASE_URL = https://github.com/Mellanox/SAI-Implementation/raw/d5810ae692f47898bbf6e093875c89b5f7e4ef19/sdk
 MLNX_SDK_VERSION = 4.3.1634
```

Comment:
isn't this to be 4.3.1636 ?

Reply:
I guess the purpose of this PR is only update firmware, but keep all other SDK components unchanged.